src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -77,6 +88,10 @@ public void readFromNBT(NBTTagCompound data) {
         if (data.hasKey("ContainerInventory")) {
             MetaTileEntityQuantumTank.legacyTankItemHandlerNBTReading(this, data.getCompoundTag("ContainerInventory"), 0, 1);
         }
+        if (isExportHatch && data.hasKey("IsLocked")) {
```

Comment:
This `hasKey()` check is unneeded. If removed, the `getBoolean("isLocked")` call will return false, and the lockedFluid will be set to null, which does not create any compatibility issues.